src/components/Auth/Login.tsx

```diff
@@ -167,19 +167,24 @@ const Login = (props: { forgot?: boolean }) => {
         }, 200);
       }
     },
+
+    //Invalid OTP error handling
     onError: (error: HTTPError) => {
-      const errorData = error.cause as { errors: Array<{ otp: string }> };
-      const errors = errorData?.errors;
-      if (Array.isArray(errors) && errors.length > 0) {
-        // BE returns a different format for invalid OTP
-        // TODO: fix this
-        //const firstError = errors[0] as OtpError;
-        //setOtpError(firstError.msg);
-        const firstError = errors[0];
-        setOtpValidationError(firstError.otp);
-      } else {
-        setOtpValidationError(t("invalid_otp"));
+      Notification.Error({ msg: t("invalid_otp") });
```

Comment:
the otp error notification is likely not being dispatched due to it's useQuery being passed with silent=true.

This is ideally auto handled by our global error handler defined here:

https://github.com/ohcnetwork/care_fe/blob/develop/src/Utils/request/errorHandler.ts

As for the set otp error for the field error, LGTM.